lib/pagseguro/subscription_change_status/response.rb

```diff
@@ -0,0 +1,26 @@
+module PagSeguro
+  class SubscriptionChangeStatus
+    class Response
+
+      attr_reader :response
+      attr_reader :object
+
+      def initialize(response, object)
+        @response = response
+        @object = object
+      end
+
+      def serialize
+        unless success?
+          object.errors.add response
+        end
+
+        object
+      end
+
+      def success?
+        response.xml? && response.success?
```

Comment:
Any reason to use `response.xml? && response.success?` instead of `response.success? && response.xml?`?

Remember that we have tasks to reduce duplicity of code… this line in this order can mean something :P